docs/mddocs/Quickstart/ollama_portablze_zip_quickstart.md

```diff
@@ -79,4 +84,49 @@ To increase the context length, you could set environment variable `IPEX_LLM_NUM
 - Start Ollama serve through `start-ollama.bat`
 
 > [!TIP]
-> `IPEX_LLM_NUM_CTX` has a higher priority than the `num_ctx` settings in a models' `Modelfile`.
\ No newline at end of file
+> `IPEX_LLM_NUM_CTX` has a higher priority than the `num_ctx` settings in a models' `Modelfile`.
+
+## Linux Quickstart
+
+### Linux Prerequisites
+
+Check your GPU driver version, and update it if needed:
+
+- For client GPU, like A-series, B-series and integrated GPU, we recommend following https://dgpu-docs.intel.com/driver/client/overview.html to install your GPU driver.
+- For Data Center GPU, like Max series and FLEX series, we recommend following https://dgpu-docs.intel.com/driver/installation.html to install your GPU driver.
+
+### Linux Step 1: Download and Extract
+
+Download IPEX-LLM Ollama portable tgz from the [link](https://github.com/intel/ipex-llm/releases/tag/v2.2.0-nightly).
+
+Then open a terminal, extract the tgz file to a folder.
+```
+tar xf [Download tgz file]
```

Comment:
May use something like `tar xf PATH/TO/DOWNLOADED/TGZ` as we did for `cd PATH/TO/EXTRACTED/FOLDER`?